TCMSApp/src/client/app/admin/team.controller.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @ngdoc controller
+ * @name teamController
+ * @memberOf app.admin
+ * @description
+ *  Control Manage Team template. Add, delete, edit users in the current project. Synchronize with Trello
+ */
+
+(function () {
+    'use strict';
+
+    angular
+        .module('app.admin')
+        .controller('TeamController', TeamController);
+
+    TeamController.$inject = ['logger', 'Trello', 'TeamMembersFactory'];
+
+    function TeamController(logger, Trello, TeamMembersFactory) {
+        var teamVm = this;
+        teamVm.users = [];
+        teamVm.organization = "test04498212";
+
+        teamVm.getUsers = TeamMembersFactory.getUsers;
+
+        teamVm.addUser = addUser;
+        teamVm.deleteUser = TeamMembersFactory.deleteUser;
+
+
+        activate();
+        setDefaultInput();
+        TeamMembersFactory.auth();
+        getMe();
```

Comment:
why getMe is here and everything else that we use to work with trello is in TeamMembersFactory?